lib/services/cloud-service.ts

```diff
@@ -97,4 +116,30 @@ export abstract class CloudService extends EventEmitter {
 
 		return targetFileNames;
 	}
+
+	protected async getCollectedLogs(serverResponse: IServerResponse): Promise<string> {
```

Comment:
I think you should move this to CloudOperationV1 - doesn't it apply only to v1 operations?

Reply:
Edit: Move to cloud-operation-base

Reply:
Done.